### DISCUSSION TITLE
Budgie desktop vanished

### DISCUSSION THREAD

**Discussion by Del6747432** [bareboat-necessities/lysmarine_gen#218] [Q&A] [answered]
Switched on Raspberry today, and the start button ( top left) and the other icons at top right have vanished ??

I still have the centre top button for desktop, and far right clock and calender. But that's all there is on the top taskbar.

Tried switching on and off etc etc, but no joy 

I can only access the programs that have icons on the 2 desktops, which includes terminal.

Is there a command via terminal I can use to restart this?

**Comment by mgrouch** [comment, marked as answer]
There is ‘Commands’ icon on desktop 
and there is option to restart desktop in it

You can also launch budgie desktop from terminal 


````

budgie-desktop 

````

**Reply by Del6747432** [reply]
Thanks,

The command didn't work from the command icon, I had to use terminal.

Now, it seems, there is 2 top desktops, on top of each other ? Lol .

The first one with the problem, with no start button etc and now when I use that command, the proper working desktop shows above the other at the top.

I also need to use that command everytime I reboot the pi .

**Reply by mgrouch** [reply]
Which version are using? 

````
cat /etc/bbn-build
````

**Reply by Del6747432** [reply]
2022-07-14

It was working fine, just one day, switched on and it had messed up

**Reply by mgrouch** [reply]
Looks like some change prior to that bad boot is what caused it. Easier probably to reinstall the whole thing by burning SD card with 
an image again.

**Reply by Del6747432** [reply]
Yep, I think so. A job for tonight.

For future build releases , do I just update and upgrade at command point? Or do I need to download and burn a SD card?
